### ISSUE TITLE
[Suggestion] Custom xp drop colours for each skill

### ISSUE THREAD

**Issue by korvl38** [l2-/template-plugin#104]
I notice the "XP Globes" plugin has default progress bar colours for each skill - fishing is blue, herblore is green etc. Was hoping this plugin could borrow the same idea and allow to customize every for some extra spice. Right now there's only a single colour for all xp drops aside from offensive prayers. Cheers

**Comment by l2-** [comment]
The idea is cool but I don't want to do this right now. Can't think of a intuitive way to allow a player to configure this. Plus this plugin already has an absurd amount of settings and I don't really play the game actively at the moment. But i 'll leave it open for later

**Comment by korvl38** [comment]
@l2- yeah no worries I liked that there's a fuckton of settings so thought it'd fit right in. As a user only i don't like plugins that are barebones. I just imagine all the skills on individual rows with a box to open colour selector, default skill colours included, maaaaybe individual font selection if you toggle off global font settings idk. thanks for leaving it open
